js/i18n.js

```diff
@@ -239,18 +244,17 @@ function languageSelectorTimeout(textBox) {
   var $textBox = $(textBox);
   var textVal = $textBox.val();
 
-  // don't search for nothing
-  if (textVal.length < 2) return;
-
   var lastSearch = $textBox.attr('data-last-search');
 
   // limit the search to the first 4 characters
   var thisSearch = (textVal.length > 4) ? textVal.substr(0, 4) : textVal;
+  $textBox.attr('data-last-search', thisSearch);
```

Comment:
This was something I complained about on the last PR, but got push back that this was the way it's always done to minimize repeat traffic.  

I think this is a better solution.